containers/python/noop_container.py

```diff
@@ -9,20 +9,20 @@ class NoopContainer(rpc.ModelContainerBase):
     def __init__(self):
         pass
 
-    def predict_ints(self, inputs):
-        return np.array([np.sum(x) for x in inputs], dtype='float32')
+    def predict_ints(self, input_item):
+        return str(sum(input_item))
```

Comment:
We UTF-8 encode them before serialization. See `PredictionResponse#add_output` in `rpc.py`.